pkg/KubeArmorOperator/internal/controller/resources.go

```diff
@@ -328,6 +354,14 @@ func deploySnitch(nodename string, runtime string) *batchv1.Job {
 								Name:      "seccomp-path",
 								MountPath: "/var/lib/kubelet/seccomp",
 							},
+							{
```

Comment:
snitch has a command line flag as well to enable OCI Hook, let's handle that as well.

Reply:
this is not being handled as of now.